Goal App/Controllers/HomeViewController.swift

```diff
@@ -45,20 +45,12 @@ class HomeViewController: UIViewController {
 
     private func getGoalsList() {
         showSpinner()
-        firestoreUtil.getGoals { [weak self] response in
-            switch response {
-            case let .success(goalList):
-                self?.originalGoalsList = goalList
-
-            default:
-                self?.originalGoalsList = []
-            }
-            DispatchQueue.main.async { [self] in
-                self?.activeGoals = self?.originalGoalsList?.filter { task in task.completed == false }
-                self?.goalTasks.reloadData()
-                self?.progressCircleSetup()
-                self?.stopSpinner()
-            }
+        Task {
+            originalGoalsList = await firestoreUtil.getGoals()
```

Comment:
`originalGoalsList` observe the last two words `Goals` and `List` the meaning of them is that its plural and not singular so you can get rid of the `List` and just name this to `originalGoals` or just `goals`

just curious, if this is `originalGoals` then is there a `duplicateGoal` somewhere?